src/ECER.Resources.Documents/Applications/IApplicationRepository.cs

```diff
@@ -84,3 +98,10 @@ public record CharacterReference(string? FirstName, string? LastName, string? Ph
 {
   public string? Id { get; set; }
 }
+
+public record CharacterReferenceSubmissionRequest(CharacterReferenceContactInformation ReferenceContactInformation, CharacterReferenceEvaluation ReferenceEvaluation, bool ResponseAccuracyConfirmation)
```

Comment:
This is good, you don't have to use request/response pattern here though. Submit(ApplicationReference) and abstract ApplicationReference would work just fine